packages/experimental/TabList/src/Tab/Tab.tsx

```diff
@@ -59,11 +62,15 @@ export const Tab = compose<TabType>({
 
       const { icon, tabKey, ...mergedProps } = mergeProps(tab.props, final, { accessibilityLabel: final.accessibilityLabel || text });
```

Comment:
I can't seem to set a custom accessibilityLabel on individual tabs (it defaults to text). Could you check to see if it works?

Reply:
Yeah this will only catch the case where accessibilityLabel is passed as a prop to an inner render of a component using Tab. It won't catch accessibilityLabel passed in as part of normal use. Probably what you want to do is merge the props first, and then set accessibilityLabel using the result of the merge